tests/foreman/cli/test_errata.py

```diff
@@ -1213,35 +1204,34 @@ def errata_host(
             'content-view-id': cv.id,
         }
     )
-    rhel8_contenthost.register(
+    rhel_contenthost.register(
         activation_keys=new_module_ak.name,
         target=target_sat,
         org=org,
         loc=None,
     )
-    rhel8_contenthost.add_rex_key(satellite=target_sat)
-    assert rhel8_contenthost.subscribed
-    assert rhel8_contenthost.applicable_errata_count == 0
+    rhel_contenthost.add_rex_key(satellite=target_sat)
```

Comment:
We use `setup_remote_execution=True`  in register() as default, so this step can be skipped as already done?
```suggestion
```